src/client.h

```diff
@@ -427,8 +429,9 @@ protected slots:
 
     void CLChannelLevelListReceived ( CHostAddress InetAddr, CVector<uint16_t> vecLevelList );
 
+    void Connecting ( QString strServerName );
     void Disconnected();
-    void SoundDeviceChanged ( QString strError );
+    void SoundDeviceChanged();
```

Comment:
I don't feel strongly about removing the error message handling. Could be reverted.

Reply:
As above:
- split this out to a separate PR
- have two slots, one for GUI, one for headless, and pass the error to each